superset-frontend/src/dashboard/components/Header/HeaderActionsDropdown/HeaderActionsDropdown.test.tsx

```diff
@@ -198,3 +198,12 @@ test('should show the properties modal', async () => {
   userEvent.click(screen.getByText('Edit dashboard properties'));
   expect(editModeOnProps.showPropertiesModal).toHaveBeenCalledTimes(1);
 });
+
+test('should only apply changes in the properties modal', async () => {
```

Comment:
I was confused by the name of the test and what it's actually doing. It's only checking that there's a button called Apply but it's not checking if the changes are only being applied and not saved. Maybe rename to 'should display the Apply button when opening the modal'?